README.md

```diff
@@ -170,6 +170,51 @@ import { Agent } from '@newrelic/browser-agent/loaders/agent'
 import { Metrics } from '@newrelic/browser-agent/src/features/metrics'
 ```
 
+## Utilizing pre-defined bundles
+
+The npm package contains our pre-defined bundles within the `/dist/cdn/` directory. This allows you to use the npm package while still importing a pre-built bundle into your HTML files. The npm package contains all our bundle files but you really only need to host and include one of the agent loaders. The loader will still load all async bundles from the New Relic CDN.
+
+```html
+<!-- Copy the configuration block from the snippet in New Relic -->
+<script>
+  ;window.NREUM||(NREUM={});NREUM.init={ ... };
+
+  ;NREUM.loader_config={ ... };
+  ;NREUM.info={ ... };
+</script>
+
+<!-- Include the lite agent -->
+<script src="/node_modules/@newrelic/browser-agent/dist/cdn/nr-loader-spa-1.x.x.min.js"></script>
+<!-- Include the pro agent -->
+<script src="/node_modules/@newrelic/browser-agent/dist/cdn/nr-loader-full-1.x.x.min.js"></script>
+<!-- Include the pro+spa agent -->
+<script src="/node_modules/@newrelic/browser-agent/dist/cdn/nr-loader-rum-1.x.x.min.js"></script>
+```
+
+The version number in the file name will change depending on the version of the npm package that is installed. It is recommended that you setup your build tooling to copy the correct file from the `/node_modules/@newrelic/browser-agent/dist/cdn/` directory into a `static` directory where you normally host static files. You can rename the file at the same time to make the `script` element static but remember to manage cache busting for your users.
+
+Also included in the bundle are un-minified and polyfilled versions of each of the agents. The polyfilled versions of the agents are built to ES5 standards. When using the polyfilled versions of the agents, remember to either include your own polyfills in the HTML above the loader `script` or use our `polyfills` script.
```

Comment:
Doing a cdn prod build, the `/dist/cdn` folder is 10.2 MB. Is this all going into our current npm pkg, which is only 1.7 MB? Any downside aside from devs having to download more bytes on their machines?
Is there a way to only include the loaders without the other chunks?

Reply:
There's always a way but I don't really think its necessary or an issue.